cmd/promtool/main.go

```diff
@@ -36,17 +36,15 @@ import (
 	"github.com/prometheus/client_golang/api"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/testutil/promlint"
-	config_util "github.com/prometheus/common/config"
+	dto "github.com/prometheus/client_model/go"
+	promconfig "github.com/prometheus/common/config"
```

Comment:
Traditionally, we put named imports in a separate block (like it was before, just that the config_util import was in the wrong block).
As it was that way before, let's keep doing so here.

Reply:
Aha, got it.

Reply:
Done, except I move `"github.com/prometheus/common/expfmt"` to the normal block as it's not named.